src/utils/constant.ts

```diff
@@ -0,0 +1,3 @@
+export const STORAGE_KEYS = {
+  THEME_TYPE: "themeType",
```

Comment:
worth it to make this a ENUM with values `SYSTEM` `DARK` and `LIGHT`?

Reply:
I guess we can but not sure if it is necessary for now. This one is extracted because it is used in multiple locations.